.github/workflows/mac.yaml

```diff
@@ -0,0 +1,21 @@
+name: macos
+
+on: [pull_request]
+
+jobs:
+  build:
+    runs-on: macos-13
+    steps:
+      - uses: actions/checkout@v4
+      - run: |
+          cd src/macos/ATDriverGenericMacOS
+          xcodebuild
+      - run: |
+          pluginkit -e use -i com.bocoup.ATDriverGenericMacOS.ATDriverGenericMacOSExtension
+          defaults write com.apple.Accessibility SpeechVoiceIdentifierForLanguage '{2 = {en = "com.bocoup.ATDriverGenericMacOS.ATDriverGenericMacOSExtension.ATDriverGenericMacOSExtension";};}'
+          say hello
+          say hello
+          say hello
+          say hello
+          say hello
+          log show --process ATDriverGenericService | tail -n 10
```

Comment:
I expect to see five occurences of the string "hello" in the service logs, but [the latest build only reported an error, "no such file or directory"](https://github.com/w3c/aria-at-automation-driver/actions/runs/7496544862/job/20408743055).

If the log really is supposed to prove that the speech was captured, then we can make the results easier to interpret with much more effort by choosing different strings: using five distinct strings will make sequence errors apparent and make it easier to detect when messages are dropped/duplicated. For instance

    say hello one of five
    say hello two of five
    say hello three of five
    say hello four of five
    say hello five of five

Reply:
This combination of `say hello` and `log show` here is to try and get any output to indicate the extension and service working.

I first tried it with one `say hello`. `log` didn't show anything.

I think the system picking up the configuration change isn't instant. It may be polling based or something?

Either way I think a next step is to use cli to listen for speech from the application extension, and repeat `say hello` until output shows up. I suspect there isn't a fixed number of `say hello` we would need to see it show up. In a way I got lucky that 5 worked, but maybe it sometimes need 10, and sometimes 1 may be enough.